camayoc/tests/qcs/test_profiles.py

```diff
@@ -61,47 +67,96 @@ def test_update(shared_client, cleanup, scan_host):
            the data
     :expectedresults: The newtork profile entry is created and updated.
     """
-    cred = HostCredential(client=shared_client, password=str(uuid4()))
+    cred = HostCredential(client=shared_client, password=uuid4())
     cred.create()
+    cleanup.append(cred)
     profile = NetworkProfile(
```

Comment:
Refactored to use camayoc uuid4() and also add cred to cleanup before other possibly exception throwing activity